web/modules/custom/multistep_form/src/Form/MultistepForm.php

```diff
@@ -0,0 +1,376 @@
+<?php
+
+namespace Drupal\multistep_form\Form;
+
+use Drupal\Core\Form\FormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Url;
+
+/**
+ * Multistep form from product, delivery, and payment pages.
+ */
+class MultistepForm extends FormBase {
+
+  /**
+   * {@inheritDoc}
+   */
+  public function getFormId() {
+    return 'custom_multistep_form';
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  public function buildForm(
+    array $form,
+    FormStateInterface $form_state,
+  ): array {
+    $config = $this
+      ->config('multistep_form.multistep_form_settings_js')
+      ->get('table_steps');
+    $enabled_config = [];
+    foreach ($config as $value) {
+      if ($value['status'] == 'enabled') {
+        $enabled_config[] = $value['id'];
+      }
+    }
+    $max_step = count($enabled_config);
+
+    $form['link'] = [
+      '#type' => 'link',
+      '#title' => $this->t('Configure your form here'),
+      '#url' => Url::fromRoute('multistep_form.multistep_form_settings_js'),
+      '#attributes' => ['class' => ['button__configure-form']],
+    ];
+
+    if ($max_step == 0) {
+      $form['text'] = [
```

Comment:
I think better will be to move this to access handler
If no steps is available user shouldn't be able to access the form